### PR TITLE
WebotsJS: fix parameters bugs

### PR DESCRIPTION
**Pull request by BenjaminDeleze**
**Description**

- Fix viewpoint when there is no boundingSphere
- Fix wrong defaultValues in jointWithBacklash MFNode
- Fix `toJS` method when the node is a baseNode and not a proto

Test it here: https://testing.webots.cloud/run?version=R2023b&url=https://github.com/cyberbotics/webots/blob/fix-web-joint-with-backlash/projects/joints/protos/Hinge2JointWithBacklash.proto&type=undefined

Note that if you try to remove one of the jointsParameters it will break the proto. However it comes from the proto construction.

### CODE REVIEW

**Comment by BenjaminDeleze** [comment]
Fix a bug where two `ifs` have been converted to an `if, else`

**Review by ad-daniel — CHANGES_REQUESTED** [review]
Thank you
When I add a robot to the endPoint (say Crazyflie), and switch to the Devices tab, several errors related to the bounding sphere are thrown when hovering above some of the devices (gps, gyro, ...)

Similarly, in the Joints tab, everything seems to be working except the root joint (for crazyflie is crazyflie, for nao is nao)

**Comment by BenjaminDeleze** [comment]
Okay I have lost some hairs with those ones:
I found several more bugs:
  - [x] I was calling ._boundingSphere directly instead of the .boundingSphere function. And the ._boundingSphere is not present in all objects
  - [x] When adding objects, the boundingSphere was not recalculated correctly. The solution I found is to recall postFinalize in order to trigger computation.
    It seems a bit overkill, but we will do it only once for each device so I think it is acceptable. I tried to use a modified version of the `recomputeBoundingSphere` function but it is a bit of a mess as it does not exist in all the classes.
  - [x] Some joints were not working because `_endPointZeroRotation` and `_endPointZeroTranslation` were not defined in the case when no endPoint was present at the node creation.
  - [ ] The anchor seems to "jump" in some special case when we change the endPoint.

File: resources/web/wwi/nodes/utils/WbBoundingSphere.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import WbVector3 from './WbVector3.js';
 import {findUpperTransform} from './node_utilities.js';
+import WbWorld from '../WbWorld.js';
```
Comment (ad-daniel) [inline resources/web/wwi/nodes/utils/WbBoundingSphere.js:3]:
```suggestion
```

Comment (ad-daniel) [reply resources/web/wwi/nodes/utils/WbBoundingSphere.js:3]: doesn't appear to be used

**Review by ad-daniel — CHANGES_REQUESTED** [review]
Nice, seems to work well, thank you!

**Review by ad-daniel — APPROVED** [review]
Thank you

**Pull request merged**